server/src/Domain/FairCalendar/Event.entity.ts

```diff
@@ -8,6 +8,7 @@ export enum EventType {
   SUPPORT = 'support',
   DOJO = 'dojo',
   FORMATION_CONFERENCE = 'formationConference',
+  ADMIN = 'admin',
```

Comment:
```suggestion
  MANAGEMENT = 'management',
```

Reply:
En anglais "gestion" peut aussi se traduire par "business administration", et parler d'administration plutôt que de management (qui fait plutôt référence à la théorie du management par ex) me semble plus approprié

On peut utiliser un terme plus long :

```suggestion
  COOP_ADMINISTRATION = 'COOP_ADMINISTRATION',
```